### PR TITLE
Add module to iterate on selection

### PR DESCRIPTION
**Pull request by Viggor**
Add module to iterate on selection fields.

### CODE REVIEW

File: selection_iteration/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+#
+#   You should have received a copy of the GNU Affero General Public License
+#   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+###############################################################################
+
+{
+    'name': 'Selection iteration',
+    'version': '0.0.1',
+    'author': 'Akretion',
```
Comment (gurneyalex) [inline selection_iteration/__openerp__.py:26]: Please add "Odoo Community Association (OCA)"

File: selection_iteration/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    - The model must inherit from 'abstract.selection.rotate'\n
+    - override _get_values_from_selection method to match with
+       your selection field \n
+    - Just create a button like:\n
+    <button name="iter_selection_next" string="Next Value"
+       context="{'selection_field': '$selection_field'}" type="object"
+       icon="gtk-go-forward"/>
+
+    For more information see the example in the module.
+    """,
```
Comment (gurneyalex) [inline selection_iteration/__openerp__.py:41]: Can you extract the description to README.rst and use the template https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

**Comment by gurneyalex** [comment]
@Viggor are you still interested in this PR?

@sebastienbeau if you're around, can you ping Viggor?

**Comment by gurneyalex** [comment]
needs squashing and rebasing.

File: selection_iteration/__openerp__.py (hunk 2 above)
Comment (dreispt) [inline selection_iteration/__openerp__.py:41]: I would love to see here a real life use case example.

**Comment by gurneyalex** [comment]
No actions from author. I'm closing the PR. 

This can be reopened if the work restarts (maybe on a more recent version of Odoo.)

**Pull request closed without merging**